ignition/examples.md

```diff
@@ -65,6 +67,57 @@ data and refuse to overwrite it.
 }
 ```
 
+### XFS ###
+
+```json
+{
+	"ignitionVersion": 1,
+	"storage": {
+		"filesystems": [
+			{
+				"device": "/dev/disk/by-label/ROOT",
+				"format": "xfs",
+				"create": {
+					"force": true,
+					"options": [
+						"-L", "ROOT"
+					]
+				}
+			}
+		]
+	}
+}
+```
+
+The create options are forwarded as-is to the underlying mkfs.$format
+utility, their respective man pages document what's appropriate here.
+
+## Create Files on the Root Filesystem ##
+
+Whether formatting a new filesystem or reusing the existing one, files may be
+created in the filesystem on the named device.  When not also formatting, care
+must be taken to specify the appropriate format.
```

Comment:
When using an existing, already-formatted filesystem, be sure to supply its filesystem format (e.g., `ext4`, `btrfs`) in the configuration used to create files.